src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -137,29 +114,28 @@ private ProjectInfo CreateCsxProject(string csxPath, IScriptServicesBuilder base
                 return Context.CsxFileProjects[csxPath];
             }
 
-            // Process the file with ScriptCS first
             Logger.LogInformation($"Processing script {csxPath}...");
             Context.CsxFilesBeingProcessed.Add(csxPath);
-            var localScriptServices = baseBuilder.ScriptName(csxPath).Build();
-            var processResult = localScriptServices.FilePreProcessor.ProcessFile(csxPath);
+
+            var fileParser = new FileParser(Context.RootPath);
```

Comment:
Right I see what you mean now. Of course, let me pluck the naive syntax parsing :+1: